src/main/java/de/rabea/Router.java

```diff
@@ -0,0 +1,22 @@
+package de.rabea;
+
+import de.rabea.controller.NotFoundController;
+import de.rabea.request.HttpRequest;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Router {
+    private Map<String, Controller> configuredRoutes = new HashMap<>();
+
+    public Controller getController(HttpRequest request) {
+        if (configuredRoutes.containsKey(request.requestLine().uri())) {
```

Comment:
You can rewrite this as `configuredRoutes.getOrDefault(key, notFoundController())`